doc/book/admin/start_stop_instance.rst

```diff
@@ -3,335 +3,387 @@
 
-Starting instances
-------------------
+.. _configuration_run_instance:
 
-All the instance files or directories placed in the ``instances_enabled`` directory
-specified in :ref:`tt configuration <tt-config_file>` are called *enabled instances*.
-If there are several enabled instances, ``tt start`` starts a separate Tarantool
-instance for each of them.
+Starting Tarantool instances
+----------------------------
 
-Learn more about working with multiple Tarantool instances in
-:ref:`Multi-instance applications <admin-start_stop_instance-multi-instance>`.
+.. _configuration_run_instance_tt:
 
-To start a specific enabled instance, specify its name in the ``tt start`` argument:
+Starting instances using the tt utility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: console
+The :ref:`tt <tt-cli>` utility is the recommended way to start Tarantool instances.
 
-    $ tt start my_app
-       • Starting an instance [my_app]...
+.. code-block:: console
 
-When starting an instance, ``tt`` uses its :ref:`configuration file <tt-config>`
-``tt.yaml`` to set up a :ref:`tt environment <tt-cli-environments>` in which the instance runs.
-The default ``tt`` configuration file is created automatically in ``/etc/tarantool/``.
-Learn how to set up a ``tt`` environment in a directory of your choice in
-:ref:`Running Tarantool locally <admin-start_stop_instance-running_locally>`.
+    $ tt start sharded_cluster
+       • Starting an instance [sharded_cluster:storage-a-001]...
+       • Starting an instance [sharded_cluster:storage-a-002]...
+       • Starting an instance [sharded_cluster:storage-b-001]...
+       • Starting an instance [sharded_cluster:storage-b-002]...
+       • Starting an instance [sharded_cluster:router-a-001]...
 
-After the instance has started and worked for some time, you can find its artifacts
+After the cluster has started and worked for some time, you can find its artifacts
 in the directories specified in the ``tt`` configuration. These are the default
-locations:
+locations in the local :ref:`launch mode <tt-config_modes>`:
+
+*   ``sharded_cluster/var/log/<instance_name>/`` -- instance :ref:`logs <admin-logs>`.
+*   ``sharded_cluster/var/lib/<instance_name>/`` -- :ref:`snapshots and write-ahead logs <concepts-data_model-persistence>`.
+*   ``sharded_cluster/var/run/<instance_name>/`` -- control sockets and PID files.
+
+In the system launch mode, artifacts are created in these locations:
+
+*   ``/var/log/tarantool/<instance_name>/``
+*   ``/var/lib/tarantool/<instance_name>/``
+*   ``/var/run/tarantool/<instance_name>/``
+
+
+.. _configuration_run_instance_tarantool:
+
+Starting an instance using the tarantool command
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``tarantool`` command provides additional :ref:`options <configuration_command_options>` that might be helpful for development purposes.
```

Comment:
Maybe highlight somehow that `tarantool` run _only_ specific instances?

Reply:
I think that the next sentence should give this understanding:

> Below is the syntax for starting a Tarantool instance configured in a file: